### ISSUE TITLE
Wrong reference genome indicated in the juicer.log

### ISSUE THREAD

**Issue by Audald** [theaidenlab/juicer#364]
Hi there,

I am running Juicer as follows (as reported by the `aligned/header` file):

`Experiment description: Juicer version 1.6; BWA 0.7.17-r1188; 20 threads; openjdk version "11.0.19" 2023-04-18; /opt/scripts/juicer.sh -d /path/to/folder -D /opt/ -p /path/to/folder/ref/clean.chrom.sizes -y /path/to/folder/ref/sites.txt -z /path/to/folder/ref/genome.fasta -t 20`

However, the `juicer.log` indicates that I am aligning my reads against the `hg19` genome.

```
(-: Looking for fastq files...fastq files exist
(-: Aligning files matching /path/to/folder/fastq/*_R*.fastq* to genome hg19 with site file /path/to/folder/ref/sites.txt
(-: Created /path/to/folder/splits and /path/to/folder/aligned.
```
I suspect this is a bug/error (confirmed by the [Arima Technical Support Team](https://github.com/ArimaGenomics/Juicer_pipeline_containers/issues/1)) because when I try to force the mapping to `hg19` (instead of my `-z genome.fasta`) with `-g hg19`:

`Reference sequence /opt//references/homo_sapiens_assembly19.fasta does not exist` (as expected)

Just raising this bug in case you want to correct the way the log files are being reported. Best!

**Comment by lscnsdad** [comment]
hi，How to solve this problem？

**Comment by Audald** [comment]
I don't think there is a fix yet. It seems to be a reporting problem but hopefully does not impact the results.

**Comment by PhyloJuice** [comment]
use -g your_genome can help you.

**Comment by Audald** [comment]
> use -g your_genome can help you.

`-z` seems to be the [correct flag](https://github.com/aidenlab/juicer?tab=readme-ov-file#quick-start) when the reference genome file is provided. As far as I understand `-g` is adequate for widely used reference genomes (`hg19` or `mm10`).

**Comment by PhyloJuice** [comment]
yeah, it is not impact the results. -g only alter the name of your genome in the log file but only -z can provide reference genome
